barman/utils.py

```diff
@@ -626,8 +626,8 @@ def check_tli(value):
         return value
     try:
         return check_positive(value)
-    except ArgumentTypeError:
-        raise ArgumentTypeError("'%s' is not a valid TLI" % value)
+    except ArgumentTypeError as err:
+        raise ArgumentTypeError("%s" % err)
```

Comment:
Since we're now just re-raising the original exception we can do away with the `try/except` block altogether.